src/pages/ScholarshipForm.jsx

```diff
@@ -25,43 +68,70 @@ function ScholarshipForm() {
     setFormFieldStates({ ...formFieldStates, minAmount, maxAmount });
   }
 
-  function requiredTextField(id, label) {
-    return (
-      <TextField
-        id={id}
-        label={label}
-        required
-        fullWidth
-        value={formFieldStates[id]}
-        onChange={updateFn(id)}
-      />
-    );
-  }
-
   return (
     <Container maxWidth="md">
       <form>
         <h1>Submit a Scholarship</h1>
-        {requiredTextField('name', 'Scholarship Name')}
+        <div>
+          <TextField
+            id="name"
+            label="Scholarship Name"
+            error={formFieldErrors.name}
+            helperText={formFieldErrors.name ? 'Please enter a name' : ' '}
+            className={classes.textFieldStyle}
+            required
+            value={formFieldStates.name}
+            onChange={updateFn('name')}
+          />
+        </div>
         <DatePicker
           id="deadline"
           label="Deadline"
+          error={formFieldErrors.deadline}
+          helperText="Please enter a valid date"
           value={formFieldStates.deadline}
           onChange={(date) =>
             setFormFieldStates({ ...formFieldStates, deadline: date })
           }
         />
-        {requiredTextField('description', 'Description')}
-        {requiredTextField('website', 'Website')}
+        <div>
+          <TextField
+            id="description"
+            label="Description"
+            error={formFieldErrors.description}
+            helperText={
+              formFieldErrors.description ? 'Please enter a description' : ' '
+            }
+            className={classes.textFieldStyle}
+            required
+            value={formFieldStates.description}
+            onChange={updateFn('description')}
+          />
```

Comment:
Make this [Multiline](https://material-ui.com/components/text-fields/#multiline)